coast/cdf.py

```diff
@@ -4,7 +4,7 @@
 from .logging_util import get_slug, debug, error
 
 
-class CDF:
+class Cdf:
```

Comment:
I guess this may just be a preference thing but could Cdf not stay capitalised as it is an acronym? 
PEP8 sort of mentions this too.

![image](https://user-images.githubusercontent.com/64262641/134022080-a472ceee-4416-447d-bc45-a871a884a455.png)

Reply:
@Fakas Have made this change in the most recent push. Seems this class isn't used anywhere else in the codebase, so it was literally just 2 characters altered.